test/utilities/fileSystem.ts

```diff
@@ -4,37 +4,63 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { ChildProcess, exec } from 'child_process';
-import * as fs from 'fs-extra';
+import * as fs from 'fs';
 import path from 'path';
 import { TestSetup } from '../testSetup.ts';
 import { log } from './miscellaneous.ts';
+import FastGlob from 'fast-glob';
 
-export function createFolder(folderPath: string): ChildProcess {
-  const childProcess = exec(`mkdir "${folderPath}"`);
-
-  return childProcess;
+export function createFolder(folderPath: string): void {
```

Comment:
Got rid of fs-extra

Reply:
Does this mean we can remove `"@types/fs-extra": "11.0.4",` from package.json?  `fs-extra` is not used anywhere else in the codebase.

Reply:
I'll check for other references to the module and remove fs-extra, if possible.